how-tos/fennec-temp-relpro.md

```diff
@@ -0,0 +1,44 @@
+# Forewords
+
+These steps are meant to be specifically for Fennec-58.0bN.
+
+# High-level steps
+
+1. ship-it
+1. steps after QA signed off
+
+# Detailed descriptions
+
+## ship-it
+
+Ship-it should be hooked up as normal, using release-runner3.
+
+### Relpro action task
+
+There will be a "Relpro" action task that shows up in Treeherder. This will be part of the on-push graph, but it will generate a new Promotion graph. The promotion graph's taskGroupId will be the same as the action task's taskId. This graph will contain all the promotion tasks that are needed to build and push Fennec to candidates.
```

Comment:
font nit: might be worth including taskGroupId between `quotes` to highlight it.

Reply:
`taskid` too.